pkg/smb/fake_mounter_test.go

```diff
@@ -25,6 +25,9 @@ import (
 )
 
 func TestMount(t *testing.T) {
+	targetTest := "./target_test"
+	sourceTest := "./source_test"
```

Comment:
Whenever these tests are refactored, we'll similarly change the hard coded paths to `testutil.GetWorkDirPath`

Reply:
anything you want to do in this PR?

Reply:
Nope, I think we can merge this one. I will open a follow up PR for refactoring remaining tests on this repo, as well as on the azurefile and azure disk repos